robocorp-code/vscode-client/src/views.ts

```diff
@@ -1,12 +1,27 @@
-import { TREE_VIEW_ROBOCORP_CLOUD_TREE, TREE_VIEW_ROBOCORP_LOCATORS_TREE, TREE_VIEW_ROBOCORP_ROBOTS_TREE, TREE_VIEW_ROBOCORP_ROBOT_CONTENT_TREE } from './robocorpViews';
+import {
```

Comment:
Note: we should probably settle into some formatter for the future so that imports aren't all reformatted all the time -- no need to change anything right now, just a reminder to myself while notifying you too ;)

Reply:
Right, that makes sense. Maybe we should config a prettier/linter configs for python and typescript/javascript as well, if not already there.. I've tried to remember to format python with black at least :)